examples/cookbook/utils/deploy-contract.js

```diff
@@ -5,7 +5,8 @@ const homedir = require("os").homedir();
 
 const CREDENTIALS_DIR = ".near-credentials";
 const ACCOUNT_ID = "near-example.testnet";
-const WASM_PATH = "./wasm-files/status_message.wasm";
+const WASM_PATH = "/wasm-files/status_message.wasm";
+const pathToWasm = path.join(__dirname, WASM_PATH);
```

Comment:
`WASM_PATH` and `pathToWasm` are a bit confusing.
Let's make it more descriptive.